### PR TITLE
Adding info about "team throttling" in about/teams

### PR DESCRIPTION
**Pull request by trebmuh**
See #1323

### CODE REVIEW

File: www/about/teams.spt [hunk 1]
```diff
@@ -129,6 +129,11 @@ title = _("Teams")
     "been created after the most recent payday)."
 ) }}</p>
 
+<p>{{ _(
+    "You can enable or not this option a the very moment you're creating a team. "
+    "If you want to change this option later, you need to contact our support."
```
Comment (mattbk) [inline www/about/teams.spt]: Include a link to email?

File: www/about/teams.spt [hunk 2]
```diff
@@ -129,6 +129,11 @@ title = _("Teams")
     "been created after the most recent payday)."
 ) }}</p>
 
+<p>{{ _(
+    "You can enable or not this option a the very moment you're creating a team. "
```
Comment (mattbk) [inline www/about/teams.spt]:
```suggestion
    "You can enable this option when you create a team. "
```

File: www/about/teams.spt (hunk 1 above)
Comment (trebmuh) [reply www/about/teams.spt]: I thought about that when writing it but I saw there is another "contact" in the "Removing team membership" of the https://liberapay.com/about/teams page, plus there is a "contact" tab at the beginning of the same page, so I ended not putting a link.

**Comment by trebmuh** [comment]
Thanks for the review and comment @mattbk .
@Changaco how does that looks for you?

File: www/about/teams.spt (hunk 1 above)
Comment (mattbk) [reply www/about/teams.spt]: Right on.

**Comment by Changaco** [comment]
I've reworded the paragraph to be more consistent with the rest of the page, please review.

**Comment by trebmuh** [comment]
LGTM

**Comment by trebmuh** [comment]
:+1:

**Pull request merged**